app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -53,6 +53,7 @@ public static SessionStore get() {
         boolean mIsInputActive;
         GeckoSession.ProgressDelegate.SecurityInformation mSecurityInformation;
         String mUri;
+        String mRegion;
```

Comment:
should I be storing the state here or elsewhere?

Reply:
Commented below, region is a app wide setting so maybe there is no need of setting it per session and just do it as a SessionStore field?